COMET.Web.Common.Tests/Server/Services/ConfigurationService/ConfigurationServiceTestFixture.cs

```diff
@@ -33,6 +34,8 @@ namespace COMET.Web.Common.Tests.Server.Services.ConfigurationService
 
     using NUnit.Framework;
 
+    using JsonSerializer = System.Text.Json.JsonSerializer;
```

Comment:
Is this using needed?

Reply:
It's not - this was a mistake when I allowed the IDE to get me the reference of the serializer.. since Newtonsoft was there first, it attempted to specify which JsonSerializer class to use. I'll update this :)